public/locales/en.json

```diff
@@ -389,11 +389,31 @@
             },
             "preferences": {
                 "title": "Preferences",
-                "theme": "Editor theme",
-                "keyMap": "Keymap",
-                "indentWithTabs": "Tab character",
+                "theme": {
+                    "label": "Editor theme",
+                    "one-dark": "Dark",
+                    "neat": "Light"
+                },
+                "keyMap": {
+                    "label": "Keymap",
+                    "sublime": "Sublime",
+                    "emacs": "Emacs",
+                    "vim": "Vim"
+                },
+                "indentWithTabs": {
+                    "label": "Tab character",
+                    "on": "Tabs",
+                    "off": "Spaces"
+                },
                 "indentUnit": "Tab size (when using spaces)",
```

Comment:
Maybe the "when using spaces" could be removed as you wrapped the quite nice show-if around this field.

Reply:
I changed the translation